scripts/pre_commit_hook.py

```diff
@@ -94,38 +92,11 @@ def _does_diff_include_package_lock_file_and_no_package_file():
 
     if not err:
         files_changed = out.split('\n')
-        return 'package-lock.json' in files_changed and (
-            'package.json' not in files_changed)
+        return 'package-lock.json' in files_changed
```

Comment:
are we still going to have `package-lock.json` file? (I see you have deleted the package-lock.json file.)

Reply:
we won’t have `package-lock.json` moving forward. Yarn uses `yarn.lock`

Reply:
yarn.lock will not change automatically, right?

Reply:
The only time it will is when there's a addition, deletion, or updating of a package. When during pure installs, the file does not change.